### PR TITLE
refactor: do not change active account to sign multisig action

### LINKED ISSUE #2868: MultiSig - Approve Transaction Modal - Show in the list account related to the approval.

### PR DESCRIPTION
**Pull request by CedrikNikita**
fixes #2868

### CODE REVIEW

File: src/composables/multisigTransactions.ts [hunk 1]
```diff
@@ -132,6 +132,7 @@ export function useMultisigTransactions() {
     action: TxFunctionMultisig,
     contractId: Encoded.ContractAddress,
     spendTxHash: string,
+    options = {},
```
Comment (peronczyk) [inline src/composables/multisigTransactions.ts:135]: I suggest to type the possible options.

File: src/popup/components/Modals/MultisigProposalConfirmActions.vue [hunk 2]
```diff
@@ -129,6 +128,8 @@ export default defineComponent({
       (): IFormSelectOption[] => prepareAccountSelectOptions(pendingMultisigTxLocalSigners.value),
     );
 
+    const chosenAccountAddress = ref(pendingMultisigTxLocalSigners.value[0].address);
```
Comment (peronczyk) [inline src/popup/components/Modals/MultisigProposalConfirmActions.vue:131]: I suggest to put references before computed properties.

File: src/popup/pages/MultisigProposalDetails.vue [hunk 3]
```diff
@@ -320,6 +321,7 @@ import {
 } from '@/protocols/aeternity/helpers';
 import { ProtocolAdapterFactory } from '@/lib/ProtocolAdapterFactory';
 
+import { AeAccountHdWallet } from '@/protocols/aeternity/libs/AeAccountHdWallet';
```
Comment (peronczyk) [inline src/popup/pages/MultisigProposalDetails.vue:324]: I suggest to add empty line before lib imports and component imports :)

**Review by peronczyk — APPROVED** [review]

**Comment by Liubov-crypto** [comment]
LGTM

**Pull request merged**